bool docking_complete
---
bool success

